packages/frama-c/frama-c.10.0/opam

```diff
@@ -66,7 +66,7 @@ patches: ["0004-Port-to-OCamlgraph-1.8.5.patch"]
 
 conflicts: [
   "why3" { >= "0.85" }
-  "ocaml" { = "4.03.0" & os-distribution = "debian" }
+  "ocaml" { = "4.03.0" & os-family = "debian" & os-distribution != "ubuntu" }
```

Comment:
This constraint seems really really weird but ok: it came from https://github.com/ocaml/opam-repository/pull/12861 and i think it's fine if it stayed with `os-distribution`

Reply:
Seeing how specific this constraint is, it's probably safe either way :-). Reverting